packages/esm-patient-registration-app/src/patient-registration/patient-registration.resource.tsx

```diff
@@ -184,9 +182,23 @@ export function usePatientPhoto(patientUuid: string): UsePatientPhotoResult {
 export async function fetchPerson(query: string) {
   const abortController = new AbortController();
 
-  return openmrsFetch(`/ws/rest/v1/person?q=${query}`, {
+  const patientsRes = await openmrsFetch(`/ws/rest/v1/patient?q=${query}`, {
     signal: abortController.signal,
   });
+
+  const personsRes = await openmrsFetch(`/ws/rest/v1/person?q=${query}`, {
+    signal: abortController.signal,
+  });
```

Comment:
Could we do this with a ` const [patientsRes, personsRes] = await Promise.all([fetch1, fetch2])`? That way we can run both fetches in actual parallel.

```
await fn();
await fn2();
```

Is semantically equivalent to:

```
Promise.resolve(fn()).then(Promise.resolve(fn2())).then(/* remainder of function */)
```